docs/ide-usage.md

```diff
@@ -88,59 +88,30 @@ The following plugins are needed:
 Supported IntelliJ versions: IntelliJ IDEA Ultimate Edition (required to work with Google App Engine).
 You can sign up for the free [JetBrains student license](https://www.jetbrains.com/student/) if you are a student registered in an educational institution.
 
-**NOTE**
-> If you are migrating from Eclipse, you should delete `.project`, `.classpath`, `.launches`,
-> `src/main/webapp/WEB-INF/classes`, `src/main/webapp/WEB-INF/lib` and `src/main/webapp/WEB-INF/appengine-generated`.\
-> You can also copy `local_db.bin` from `src/main/webapp/WEB-INF/appengine-generated` to the location your dev server in
-> IntelliJ reads the datastore from after your IntelliJ setup. See [dev server setup](development.md#with-intellij) for
-> more details.
-
 ### Prerequisites
-1. You need a Java 7 SDK with the name `1.7` defined in IntelliJ IDEA as follows:
+1. You need a Java 8 SDK with the name `1.8` defined in IntelliJ IDEA as follows:
 
     1. If you have no current projects open, click `Configure → Project Defaults → Project Structure`.\
        **OR**\
        If you currently have projects open, click `File → Project Structure`.
-    1. Select SDKs in Platform Settings and check if there is a SDK named `1.7` with a JDK home path pointing to a
-       JDK 7 path. Otherwise add a new SDK using JDK 7 with a name of `1.7`.
+    1. Select SDKs in Platform Settings and check if there is a SDK named `1.8` with a JDK home path pointing to a
+       JDK 8 path. Otherwise add a new SDK using JDK 8 with a name of `1.8`.
 
-1. You need the system property `ide=idea` for `Gradle VM options:` under `Global Gradle settings` as follows:
+1. You need the `Google Cloud Tools` plugin installed and configured:
 
-    1. If you have no current projects open, click `Configure → Settings/Preferences`.\
+    1. If you have no current projects open, click `Configure → Plugins`.\
        **OR**\
-       If you currently have projects open, click `File → Settings` or `IntelliJ IDEA → Preferences`.
-    1. Go to `Build, Execution, Deployment → Build Tools → Gradle`. Under `Global Gradle settings`,
-       add `-Dide=idea` to `Gradle VM options:`.
-
-1. You need to have an application server named `AppEngine Dev <version>` pointing to the SDK you downloaded previously as follows: 
-    
+       If you currently have projects open, click `File → Settings → Plugins` or `IntelliJ IDEA → Preferences → Plugins`.
+    1. Click on `Browse repositories...`. Select `Google Cloud Tools` and click `Install`.
+    1. During installation, you may encounter two prompts: one to install the `Google Account` plugin dependency and
+       another to disable the obsolete `Google App Engine Integration` plugin. Answer `Yes` to both prompts.
+    1. Restart IntelliJ IDEA.
     1. If you have no current projects open, click `Configure → Settings/Preferences`.\
```

Comment:
I tried just now in Mac, `IntelliJ IDEA → Preferences` is still available even when no projects open. Similarly for the step when configuring Cloud SDK directory above.

Reply:
This is not available on Windows though - `Configure` is the only way to get to the settings there.